aws-core/src/main/java/com/amplifyframework/auth/CognitoCredentialsProvider.kt

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Why revert the year? (I noticed that had happened above as well) Just a copy/paste artifact? In any case, use year-free version, or 

```suggestion
 * Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Reply:
Hmm, I may have the wrong year in my auto license config, but its odd it changed correctly on others. I'll fix all of these.